test/stages/variables.tf

```diff
@@ -78,5 +78,11 @@ variable "cp_entitlement_key" {
 }
 
 variable "backup_storage_size" {
+  type = string
   default = "2Gi"
+}
+
+variable "cp4s_admin_user" {
+  type = string
+  default = "dimallya@in.ibm.com"
```

Comment:
Should this email be left in the commit?

Reply:
I have updated the default value to example user email id now @triceam